README.md

```diff
@@ -72,12 +71,20 @@ This action is designed to be used with the settings "Allow merge commits" and "
 
 <img width="808" src="https://github.com/motlin/forbid-merge-commits-action/assets/244258/138ab7c1-6c14-4b1b-8bc4-a61fc3f6cfb6">
 
+This action **does not** enforce that your PR branch is up-to-date with the base branch. To ensure Pull Requests are up-to-date with the base branch before merging, enable GitHub's "Require branches to be up to date before merging" option in your branch protection rules.
+
+<img width="797" alt="require-branches-up-to-date" src="https://github.com/user-attachments/assets/5cfc4c72-7f5d-4f9e-b93f-704762b957c6" />
+
 ## How is this different from the "Require Linear History" status check?
 
 "Require linear history" does not allow merges at all. It is used with "Squash and Merge" or "Rebase and Merge" and creates a completely linear history.
 
 This workflow is meant to be used with the "Merge" button on Pull Requests. These merge commits let us see who clicked the merge button, and which commits were grouped together into a single Pull Request with multiple commits.
 
+## GitHub Merge Queue Compatibility
+
+GitHub's merge queue does not automatically rebase queued PRs on top of each other before merging. While this action prevents merge commits within individual PRs, the merge queue may still produce a non-linear history when multiple PRs are merged in succession.
```

Comment:
Would setting max to 1 work?

'Maximum pull requests to merge'

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue

Reply:
Got it!

If you work at a company that is paying Github for it's service, try to submit this request (semi-linear merge support) to them from the paying account (they'll listen more to paying customers).